packages/react-dom/src/__tests__/ReactDOMSelect-test.js

```diff
@@ -545,6 +545,48 @@ describe('ReactDOMSelect', () => {
     expect(node.options[2].selected).toBe(false); // gorilla
   });
 
+  it('should not fail', () => {
```

Comment:
This is very vague.

Reply:
Yeah, It's some temporary "name" until we get a full working version.
Going to change it to some concrete form 😄 

Have you taken a look on `anccestorInfo` in current case?

Reply:
What about this description: "Options with stateful children; it should change its state properly and shouldn't fail"
What do you think?